lib/microkube/webhook.rb

```diff
@@ -54,7 +54,7 @@ def update_config(service, image)
     if $?.success?
       update_config(service, image)
 
-      renderer = Microkube::Renderer.new
+      renderer = Microkube::Renderer.new(false)
```

Comment:
You don't need to pass the false here cause that's the default value

Reply:
Updated